packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -763,7 +763,7 @@ void main() {
     expect(tapCount, 3);
   });
 
-  testWidgetsWithLeakTracking('onTapOutside is called upon tap outside', (WidgetTester tester) async {
+  testWidgetsWithLeakTracking('onTapOutside is not called upon tap outside', (WidgetTester tester) async {
```

Comment:
nit: maybe this should be `onTapOutside is called upon tap outside when field is focused`. And maybe we should add a test `onTapOutside is not called upon tap outside when field is not focused` to verify the faulty behavior no longer happens.